--- Box2D-r29.js	Fri Nov 23 16:54:17 2012
+++ Box2D.js	Fri Nov 23 17:00:06 2012
@@ -19,7 +19,7 @@
 
 (function (a2j, undefined) {
 
-   if(!(Object.prototype.defineProperty instanceof Function)
+   if(!Object.defineProperty
       && Object.prototype.__defineGetter__ instanceof Function
       && Object.prototype.__defineSetter__ instanceof Function)
    {
@@ -10863,4 +10863,20 @@
 })(); //post-definitions
 var i;
 for (i = 0; i < Box2D.postDefs.length; ++i) Box2D.postDefs[i]();
-delete Box2D.postDefs;
\ No newline at end of file
+delete Box2D.postDefs;
+// Exports
+exports.Box2D = Box2D;
+exports.b2AABB = Box2D.Collision.b2AABB;
+exports.b2Body = Box2D.Dynamics.b2Body;
+exports.b2BodyDef = Box2D.Dynamics.b2BodyDef;
+exports.b2CircleShape = Box2D.Collision.Shapes.b2CircleShape;
+exports.b2DebugDraw = Box2D.Dynamics.b2DebugDraw;
+exports.b2Fixture = Box2D.Dynamics.b2Fixture;
+exports.b2FixtureDef = Box2D.Dynamics.b2FixtureDef;
+exports.b2MassData = Box2D.Collision.Shapes.b2MassData;
+exports.b2PolygonShape = Box2D.Collision.Shapes.b2PolygonShape;
+exports.b2PrismaticJointDef = Box2D.Dynamics.Joints.b2PrismaticJointDef;
+exports.b2RevoluteJointDef = Box2D.Dynamics.Joints.b2RevoluteJointDef;
+exports.b2Transform=Box2D.Common.Math.b2Transform;
+exports.b2Vec2 = Box2D.Common.Math.b2Vec2;
+exports.b2World = Box2D.Dynamics.b2World;
